config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   get "/", to: "application#welcome"
+  get "/applications/new", to: 'applications#new'
+  get "/applications/:id", to: 'applications#show'
+  post "/applications", to: "applications#create"
+
```

Comment:
Nice RESTful routing here, but per previous comment, you'd probably want to break out the adding-pet-to-app functionality with this additional route: 
`post "/applications/:application_id/pets/:pet_id", to "pet_applications#create"`